include/llvm-dialects/Dialect/Visitor.h

```diff
@@ -480,6 +530,23 @@ class VisitorBuilder : private detail::VisitorBuilderBase {
     PayloadT *self = static_cast<PayloadT *>(payload);
     (self->*fn)(*llvm::cast<OpT>(op));
   }
+
+  template <typename... OpTs>
+  static void memberFnSetForwarder(const detail::VisitorCallbackData &data,
+                                   void *payload, llvm::Instruction *op) {
```

Comment:
I think you can remove this overload: the template argument list isn't actually used anywhere.